.github/workflows/ci.yml

```diff
@@ -0,0 +1,129 @@
+name: Moodle plugin CI for master
```

Comment:
Should it be **master** here as we use **main** branch?

Reply:
Good point, fixing it